content/docs/rfcs/13/README.md

```diff
@@ -11,46 +11,53 @@ contributors:
   - Aaryamann Challani <aaryamann@status.im>
 ---
 
-This specification explains the Waku `13/WAKU2-STORE` protocol which enables querying of messages received through relay protocol and stored by other nodes. 
+# Abstract
+This specification explains the `13/WAKU2-STORE` protocol which enables querying of messages received through the relay protocol and 
+stored by other nodes. 
 It also supports pagination for more efficient querying of historical messages. 
 
 **Protocol identifier***: `/vac/waku/store/2.0.0-beta4`
 
-# Design Requirements
-Nodes willing to provide storage service using `13/WAKU2-STORE` protocol SHOULD provide a complete and full view of message history.
-As such, they are required to be *highly available* and in specific have a *high uptime* to consistently receive and store network messages. 
-The high uptime requirement makes sure that no message is missed out hence a complete and intact view of the message history is delivered to the querying nodes.
-Nevertheless, in case that storage provider nodes cannot afford high availability, the querying nodes may retrieve the historical messages from multiple sources to achieve a full and intact view of the past.
-
-The concept of "ephemeral" messages introduced in `[14/WAKU2-MESSAGE](/spec/14)` affects `13/WAKU2-STORE` as well.
-Nodes running [13/WAKU2-STORE](/spec/13) SHOULD support "ephemeral" messages as specified in [14/WAKU2-MESSAGE](/spec/14).
-Nodes running [13/WAKU2-STORE](/spec/13) SHOULD NOT store messages with the `ephemeral` flag set to `true`.
-
+## Terminology
+The term Personally identifiable information (PII) refers to any piece of data that can be used to uniquely identify a user. 
+For example, the signature verification key, and 
+the hash of one's static IP address are unique for each user and hence count as PII.
 
-# Security Consideration
+# Design Requirements
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-The main security consideration to take into account while using `13/WAKU2-STORE` is that a querying node have to reveal their content filters of interest to the queried node, hence potentially compromising their privacy.
+Nodes willing to provide the storage service using `13/WAKU2-STORE` protocol,
+SHOULD provide a complete and full view of message history.
+As such, they are required to be *highly available* and 
+specifically have a *high uptime* to consistently receive and store network messages. 
+The high uptime requirement makes sure that no message is missed out hence a complete and 
+intact view of the message history is delivered to the querying nodes.
```

Comment:
I don't think high uptime is a requirement here, the waku network is meant to be slightly ephemeral in nature.

Reply:
This is correct, it is specified as a SHOULD in this specification.
"Nodes willing to provide the storage service using `13/WAKU2-STORE` protocol,
SHOULD provide a complete and full view of message history."